accounts-db/src/append_vec.rs

```diff
@@ -265,6 +259,13 @@ pub struct AppendVec {
 
     /// if true, remove file when dropped
     remove_file_on_drop: AtomicBool,
+
+    /// Flags if the append vec is dirty or not.
+    /// Since fastboot requires that all storages are flushed to disk, be smart about it.
+    /// AppendVecs are (almost) always write-once.  The common case is that an AppendVec
```

Comment:
when are append vece not write-once?

Reply:
ugh. it does seem safe now to kill the append algorithm of ancient squash. In another pr, of course. Now that we have seen stability of skipping rewrites on mnb. There is zero chance we switch to the old algorithm.